internal/fabric/identity.go

```diff
@@ -126,15 +126,46 @@ func (w *rpcWrapper) List(res http.ResponseWriter, req *http.Request, params htt
 }
 
 func (w *rpcWrapper) Get(res http.ResponseWriter, req *http.Request, params httprouter.Params) (*identity.Identity, *restutil.RestError) {
-	result, err := w.caClient.GetIdentity(params.ByName("username"), params.ByName("caname"))
+	username := params.ByName("username")
+	result, err := w.caClient.GetIdentity(username, params.ByName("caname"))
 	if err != nil {
 		return nil, restutil.NewRestError(err.Error(), 500)
 	}
+
+	// the SDK identity manager does not persist the certificates
+	// we have to retrieve it from the identity manager
+	si, _ := w.identityMgr.GetSigningIdentity(username)
+	var ecert []byte
+	var mspId string
+	if si != nil {
+		// the user may have been enrolled by a different client instance
+		ecert = si.EnrollmentCertificate()
+		mspId = si.Identifier().MSPID
+	}
```

Comment:
do we need to log an error and return if signing identity is not retrieved. Looks like it will assign nil values later otherwise.

Reply:
If `si == nil` then the `newId.MSPID` will be set to `""` later (line 158) and `newId.EnrollmentCert` and `newId.EnrollmentCert` will both be empty byte arrays